modules/cli/src/main/java/org/apache/ignite/internal/cli/call/connect/ConnectCallInput.java

```diff
@@ -12,23 +12,54 @@ public class ConnectCallInput implements CallInput {
     @Nullable
     private final String password;
 
-    public ConnectCallInput(String url, @Nullable String username, @Nullable String password) {
+    private ConnectCallInput(String url, @Nullable String username, @Nullable String password) {
         this.url = url;
         this.username = username;
         this.password = password;
     }
 
-    String getUrl() {
+    String url() {
         return url;
     }
 
     @Nullable
-    String getUsername() {
+    String username() {
         return username;
     }
 
     @Nullable
-    String getPassword() {
+    String password() {
         return password;
     }
+
+    public static class ConnectCallInputBuilder {
```

Comment:
When we add a builder to the class we tend to create a static method `builder()` and close a builder's constructor.